.eslintrc.js

```diff
@@ -234,7 +227,6 @@ module.exports = {
 				'import/no-unresolved': 'off',
 				'import/named': 'off',
 				'@wordpress/data-no-store-string-literals': 'off',
-				'react-compiler/react-compiler': 'off',
```

Comment:
Haven't tested, but once you enable `react-hooks/react-compiler` manually above, you might need to disable it again here and below.